### PR TITLE
CB-24463 Quickstart scripts for express onboarding

### PR DESCRIPTION
**Pull request by daszabo**

### CODE REVIEW

File: aws/quickstart.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
```
Comment (jimright) [inline aws/quickstart.sh:2]: A license header is needed for each file. See for example: https://github.com/cloudera-labs/cdp-tf-quickstarts/blob/main/aws/main.tf#L1-L13.

File: aws/quickstart.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+export TF_VAR_aws_region=${1:-""}
+export TF_VAR_env_prefix=${2:-""}
+export TF_VAR_aws_key_pair=${3:-""}
+export TF_VAR_deployment_template=${4:-""}
+export TF_VAR_ingress_extra_cidrs_and_ports=${5:-""}
+
+# Install Terraform
+curl -fsSL https://releases.hashicorp.com/terraform/1.20/terraform_1.20_linux_amd64.zip -o terraform.zip
```
Comment (jimright) [inline aws/quickstart.sh:10]: This version of terraform doesn't exist (see 404 error below); current latest version is 1.7.1.

```
❯ curl -fsSL https://releases.hashicorp.com/terraform/1.20/terraform_1.20_linux_amd64.zip
curl: (22) The requested URL returned error: 404
```

File: azure/quickstart.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+export TF_VAR_azure_region=${1:-""}
+export TF_VAR_env_prefix=${2:-""}
+export TF_VAR_public_key_text=${3:-""}
+export TF_VAR_deployment_template=${4:-""}
+export TF_VAR_ingress_extra_cidrs_and_ports=${5:-""}
+
```
Comment (jimright) [inline azure/quickstart.sh:8]: No TF install here, is that expected?

File: azure/quickstart.sh [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```
Comment (jimright) [inline azure/quickstart.sh:2]: As above, license header is needed.

File: gcp/quickstart.sh [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```
Comment (jimright) [inline gcp/quickstart.sh:2]: As above, license header is needed.

File: gcp/quickstart.sh [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+export TF_VAR_gcp_project=${1:-""}
+export TF_VAR_env_prefix=${2:-""}
```
Comment (jimright) [inline gcp/quickstart.sh:4]: Missing the `TF_VAR_gcp_region` input variable.

**Review by jimright — CHANGES_REQUESTED** [review]
Hi @daszabo,

Just a few changes suggested after doing a quick review of the scripts.

Thanks,
Jim

File: azure/quickstart.sh (hunk 3 above)
Comment (balazsgaspar) [reply azure/quickstart.sh:8]: Azure has TF pre-installed in the Cloud Shell.
We can decide if we want to use that OOTB version or try to install and use a specific version that we control.

File: aws/quickstart.sh (hunk 2 above)
Comment (daszabo) [reply aws/quickstart.sh]: My bad, forgot to update this after testing

**Review by jimright — APPROVED** [review]
Looks good. Approved.

**Pull request merged**